### PR TITLE
Include arabic.js MathJax extension

### PR DESCRIPTION
**Pull request by OmarIthawi**
I've updated the PR to address few bugs and to include the updated extension.

**Note:** This is just a link to the extension. The actual extension and its tests is available in the following links:

 - The development repo: https://github.com/Edraak/arabic-mathjax-dev
 - The opensource pull request: https://github.com/mathjax/MathJax-third-party-extensions/pull/20
 - The testcases: http://edraak.github.io/arabic-mathjax-dev/testcases/
 - The [README.md file for documentation](https://github.com/Edraak/MathJax-third-party-extensions/blob/arabic-extension/arabic/README.md)
 - The plugin is also hosted on the math beta server.
 - [Configuration pull request](https://bitbucket.org/Edraak/configuration/pull-requests/30/wip-arabic-math-configuration/diff)

----

Related PRs: 
 - [#159 - Revert "Include arabic.js MathJax extension"](https://github.com/Edraak/edx-platform/pull/159)
 - [#152 - Include arabic.js MathJax extension](https://github.com/Edraak/edx-platform/pull/152)
 - [#149 - Include arabic.js MathJax extension](https://github.com/Edraak/edx-platform/pull/149)
 - [#109 - (WIP) Custom MathJax source and allow adding custom extensions](https://github.com/Edraak/edx-platform/pull/109)

### CODE REVIEW

**Comment by OmarIthawi** [comment]
This is officially not a WIP anymore, with the [configuration PR](https://bitbucket.org/Edraak/configuration/pull-requests/30/wip-arabic-math-configuration/diff) we're good to go inshaAllah.

**Comment by OmarIthawi** [comment]
@ahmedaljazzar @devalih This is officially not a WIP anymore, with the [configuration PR](https://bitbucket.org/Edraak/configuration/pull-requests/30/wip-arabic-math-configuration/diff) we're good to go inshaAllah.

File: cms/static/require-config.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ require.config({
             // so that require doesn't fall over
             "js/src/tender_fallback"
         ],
-        "mathjax": "//cdn.mathjax.org/mathjax/2.4-latest/MathJax.js?config=TeX-MML-AM_HTMLorMML-full&delayStartupUntil=configured",
+        "mathjax": "//cdn.mathjax.org/mathjax/2.6-latest/MathJax.js?config=TeX-MML-AM_HTMLorMML-full&delayStartupUntil=configured",
```
Comment (ahmedaljazzar) [inline cms/static/require-config.js]: Silly question.. why using the CDN? I noticed that the only CDN in this file is the mathjax's one.

File: cms/templates/base.html [hunk 2]
```diff
@@ -103,5 +103,10 @@
       });
     </script>
     <div class="modal-cover"></div>
+
+    ##  Although not so clean, this works fine!
+    ##  A better alternative is to include it using the `require-config.js`
+    ##  Is it really worth it? Not sure! May be it's cleaner this way!
```
Comment (ahmedaljazzar) [inline cms/templates/base.html]: - [ ] Uncomment these

File: common/static/sass/edraak/_arabic-mathjax.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+@import url('//fonts.googleapis.com/css?family=Amiri&amp;subset=arabic');
+
+// Hack #2.a: Putting the other missing configs from Hack #1 as styles
+.MathJax {
+  .mfliph {
+    display: inline-block !important;
+    font-family: inherit; /* Fix an error with the \pi and \prime */
+    -moz-transform: scaleX(-1);
+    -webkit-transform: scaleX(-1);
+    -o-transform: scaleX(-1);
```
Comment (ahmedaljazzar) [inline common/static/sass/edraak/_arabic-mathjax.scss]: the sass compiler can do this for you?

Comment (OmarIthawi) [inline common/static/sass/edraak/_arabic-mathjax.scss]: - [ ] Yep

File: cms/static/require-config.js (hunk 1 above)
Comment (OmarIthawi) [inline cms/static/require-config.js]: Because it is not useful (and could be harmful due to `collectstatic`) to put the mathjax inside the platform.

**Comment by ahmedaljazzar** [comment]
I'll merge since we have a deadline, but the font is not rendered on Safari.

**Pull request merged**